File: cliente-desktop/Código/PFC_V1/Modelo/Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using LiteDB;
using PFC_V1.Modelo;

namespace PFC_V1
{
    public class Usuario : Objeto
    {
        public string login { get; set; }
        public string senha { get; set; }
        public Pessoa pessoa { get; set; }
        public List<Conexao> conexoes { get; set; }
    }
}

File: cliente-web/src/app/_models/Usuario.ts
import { Objeto } from './Objeto';
import { Conexao } from './Conexao';
import { Pessoa } from './Pessoa';

export class Usuario extends Objeto {
    login: string;
    senha: string;
    email: string;
    // conexoes: Conexao[];
    pessoa: Pessoa;
}
